CHANGELOG.md

```diff
@@ -2,6 +2,164 @@
 
 ## [Unreleased](https://github.com/rerun-io/rerun/compare/latest...HEAD)
 
+## [0.13.0](https://github.com/rerun-io/rerun/compare/0.12.1...0.13.0) - Fast time series, improved layout editing & UI overrides - 2024-02-07
+
+### Overview & Highlights
+
+This release focuses on scalar time series -- both from a performance and UI perspectives.  
+Check out our [associated blog post](TODO) for more information.
+
+- 📈 Rerun can now visualize many time series in the kHz range in real-time:
+    - The new [query cache](https://github.com/rerun-io/rerun/pull/4856) optimizes data access, improving query performance by 20-50x
+    - [Sub-pixel aggregation](https://github.com/rerun-io/rerun/pull/4865) prevents unnecessary overdraw when rendering plots, improving rendering time by 30-120x
+    - [Points](https://www.rerun.io/docs/reference/types/archetypes/points3d), [lines](https://www.rerun.io/docs/reference/types/archetypes/line_strips3d), [arrows](https://www.rerun.io/docs/reference/types/archetypes/arrows3d) and [boxes](https://www.rerun.io/docs/reference/types/archetypes/boxes3d) all benefit from query caching too to a lesser extent, yielding 2-5x performance improvements
+- 🖼 UI overrides:
+    - The new `Scalar`, `SeriesLine` & `SeriesPoint` archetypes allow for customizing plots both at logging and visualization time
+    - Customize marker shapes, marker sizes, etc from code or directly through the UI
+    - Specify axis labels, lock axes, etc from code or directly through the UI
```

Comment:
@abey79 feels like we should call out something about container workflows? We had all of this hidden behind a feature flag before and having this on is pretty major